avm/res/compute/virtual-machine/modules/nic-configuration.bicep

```
@@ -0,0 +1,152 @@
+param networkInterfaceName string
+param virtualMachineName string
+param location string
+param tags object?
+param enableIPForwarding bool = false
+param enableAcceleratedNetworking bool = false
+param dnsServers array = []
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableTelemetry bool = true
```

```suggestion

@description('Required. Enable telemetry via a Globally Unique Identifier (GUID).')
param enableTelemetry bool
```

Would recommend to make it recommended here so that it's not forgotten in the future.